website/docs/r/automation_module.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_module"
+sidebar_current: "docs-azurerm-resource-automation-module"
```

Comment:
>docs-azurerm-resource-automation-module [](start = 18, length = 39)

You need to add this page to the ToC in [`azurerm.erb`](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb) as well.

Reply:
done